### ISSUE TITLE
As a CIF user, I want to see the differences between additional funding sources, so that I know that I made the right modifications

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-cif#1182]
#### Description:

Additional funding sources are handled as an array, which our current diff can't currently display.
To Do: figure out how to handle that, and implement it

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by Sepehr-Sobhani** [comment]
I think by tackling this card, we address this tech debt: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1536

**Comment by BCerki** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/1182?planning-poker) with Zenhub @dleard @gurjmatharu @JoshLarouche @mikevespi @pbastia @sam-warren @Sepehr-Sobhani @tmastrom

**Comment by pbastia** [comment]
Do we actually need item-by-item diff? Or should we show just both versions of the array
```
["funding 1","funding 2"] => ["funding 1","funding 3"]
```
This would be much simpler

**Comment by mikevespi** [comment]
I agree Pierre, I think showing old -> new makes sense and reduces the effort quite a bit
